source/includes/stackpath/_dns_records.md

```diff
@@ -122,6 +122,100 @@ Attributes | &nbsp;
 `created`<br/>*string* | The date on which the DNS record was created.
 `updated`<br/>*string* | The date on which the DNS record was last updated.
 
+<!-------------------- CREATE A DNS RECORD -------------------->
+
+#### Create a DNS record
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/dnsrecords?zoneId=337c7c26-31f4-4b2e-83dc-126exxxxxxWr"
+```
+> Request body example for creating a DNS record:
+
+```json
+{
+  "name": "*",
+  "type": "NS",
+  "ttl": 3600,
+  "data": "ns1.above.com.",
+  "weight": 1
+}
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "7135ae25-8488-4bc5-a289-285c84a00a84",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/dnsrecords?zoneId=:zoneId</code>
+
+Create a DNS record in a given [environment](#administration-environments).
+
+Query Params | &nbsp;
+---- | -----------
+`zoneId`<br/>*UUID* | The ID of the zone for which we want to delete the record. This parameter is required.
+
+Attributes | &nbsp;
+------- | -----------
+`name`<br/>*string* | The name of the record. Use the value "@" to denote current root domain name.
+`type`<br/>*string* | A DNS record's type. DNS record types describe the record's behavior.
+`ttl` <br/>*int* | A DNS record's time to live. A record's TTL is the number of seconds that the record should be cached by DNS resolvers. Use lower TTL values if you expect zone records to change often. Use higher TTL values for records that won't change to prevent extra DNS lookups by clients.
+`data`<br/>*string* | A DNS record's value.
+`weight` <br/>*int* | A DNS record's priority. A resource record is replicated in StackPath's DNS infrastructure the number of times of the record's weight, giving it a more likely response to queries if a zone has records with the same name and type.
+
+<!-------------------- UPDATE A DNS RECORD -------------------->
+
+#### Update a DNS record
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/dnsrecords/0a18b323-72df-447a-af81-08a2ea62re45?zoneId=337c7c26-31f4-4b2e-83dc-126exxxxxxWr&operation=edit"
+```
+> Request body example for creating a DNS record:
+
+```json
+{
+  "name": "*",
+  "type": "NS",
+  "ttl": 3600,
+  "data": "ns1.above.com.",
+  "weight": 1
+}
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "7135ae25-8488-4bc5-a289-285c84a00a84",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/dnsrecords?zoneId=:zoneId&operation=edit</code>
```

Comment:
We should not have to set the edit in the operation. It should be the default
```suggestion
<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/dnsrecords/:id?zoneId=:zoneId&operation=edit</code>
```

Reply:
19182ad